### ISSUE TITLE
Create list of pages for 2026 audit

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-frontend#6593]
## What

Create list of pages for 2026 audit

Look at previous lists, make sure all V6 changes are represented, create a list of (potentially) new pages to be created

## Who needs to work on this

Accessibility Specialist, (Delivery Manager?)

## Done when

- [ ] Thing to do

**Comment by kellylee-gds** [comment]
@selfthinker I've decoupled this from the procurement brief and marked that brief as complete. 

I've taken it off the cycle board and put it back onto the roadmap ready for when we pick this up again.

**Comment by selfthinker** [comment]
I have finally picked this up again and started by creating a [document (internal) that gives an overview of all the previous WCAG audits](https://beisgov.sharepoint.com/:w:/r/sites/GOVUKDS/Shared%20Documents/General/PLANNING/Planning_%20Accessibility/Design%20System%20WCAG%20audits.docx?d=w56b276d0a5334737a8913582242a4f30&csf=1&web=1&e=ZjsQgh) with links where to find details etc.
I also started to compare the list of pages that were previously audited.

**Comment by selfthinker** [comment]
I have created a [spreadsheet (internal) with all the pages that have been audited before](https://beisgov.sharepoint.com/:x:/r/sites/GOVUKDS/Shared%20Documents/General/PLANNING/Planning_%20Accessibility/Pages%20across%20audits.xlsx?d=wa06e8c7e222c49c88828ac6e484a00e9&csf=1&web=1&e=XYAtTp).

I then went through the changelog of govuk-frontend and noted down what was added or has changed since the last audit.
I have created another [(internal) document that lists all new or changed things to audit](https://beisgov.sharepoint.com/:x:/r/sites/GOVUKDS/Shared%20Documents/General/PLANNING/Planning_%20Accessibility/2026%20audit/Components%20to%20audit.xlsx?d=wedfb10a3a46d424cbc1fdec24afb3ef6&csf=1&web=1&e=nMjShH) and their URLs in the review app plus what we'd need to change or create on those pages. That's still work in progress but should be done by the end of the week.

**Comment by kellylee-gds** [comment]
We decided that we want to build a prototype type service to put our components into to be tested. We believe this service has value beyond just the audit, and can be used in user testing with end users.

We also agreed we would only have new components and components which have been iterated audited. And that there wasn't a need to audit the website as change has been minimal

**Comment by selfthinker** [comment]
I have written a [draft of a mock service in an internal Word document](https://beisgov.sharepoint.com/:w:/r/sites/GOVUKDS/Shared%20Documents/General/PLANNING/Planning_%20Accessibility/2026%20audit/Draft%20of%20pages%20for%20mock%20service%20for%202026%20WCAG%20audit.docx?d=w179381487f6645dd8388c4f254ea1a43&csf=1&web=1&e=SJpciu) that includes all the components that have changed or are new since the last audit.
I have also just created a new [GitHub issue for building that mock service](https://github.com/alphagov/govuk-frontend/issues/7357).

I have removed the last 'done when' box as that new GH issue replaces it. I haven't ticked the box before that because we will only have the URLs when the mock service has been built. But I have the URLs to all the example pages we have outside of that mock service.
And with that done I can close this issue.

**Comment by selfthinker** [comment]
I have used the [mock service that we built](https://github.com/alphagov/govuk-frontend/issues/7357) and made a [draft of the brief](https://beisgov.sharepoint.com/:w:/r/sites/GOVUKDS/Shared%20Documents/General/PLANNING/Planning_%20Accessibility/2026%20audit/Brief%20for%202026%20audit.docx?d=w1afc5f58432f41688613064b29e8f074&csf=1&web=1&e=0uCGpo) into which I have copied and described the pages to be tested.
